src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Semantic/Models/RazorSemanticTokensLegend.cs

```diff
@@ -10,18 +10,67 @@ namespace Microsoft.AspNetCore.Razor.LanguageServer.Semantic.Models
 {
     internal class RazorSemanticTokensLegend
     {
-        public const string RazorTagHelperElement = "razorTagHelperElement";
-        public const string RazorTagHelperAttribute = "razorTagHelperAttribute";
-        public const string RazorTransition = "razorTransition";
-        public const string RazorDirectiveAttribute = "razorDirectiveAttribute";
-        public const string RazorDirectiveColon = "razorDirectiveColon";
+        private const string RazorTagHelperElementString = "razorTagHelperElement";
+        private const string RazorTagHelperAttributeString = "razorTagHelperAttribute";
+        private const string RazorTransitionString = "razorTransition";
+        private const string RazorDirectiveAttributeString = "razorDirectiveAttribute";
+        private const string RazorDirectiveColonString = "razorDirectiveColon";
+        private const string RazorDirectiveString = "razorDirective";
+        private const string RazorCommentString = "razorComment";
+        private const string RazorCommentTransitionString = "razorCommentTransition";
+        private const string RazorCommentStarString = "razorCommentStar";
+
+        private const string OpenAngleString = "openAngle";
```

Comment:
These all plan to be contributed into the LSP platform's custom token types correct?

Reply:
Correct.